### DISCUSSION TITLE
Photoprism not showimg log-in screen after update

### DISCUSSION THREAD

**Discussion by MichaelLurie** [photoprism/photoprism#3465] [Q&A]
Hi
I use Portainer to manage my docker apps and I used the function "Recreate with forced pull" to update to the latest version.
Since doing so the server does not respond anymore  with error "This page can not be accessed".

In porainer it looks like everything is fine and working.

Was there a significant change recently or what can I do to try to find the source of the problem ?

Thank you in advance
Michael

**Comment by lastzero** [comment]
You may just have to wait. To shorten the startup time, do not set `PHOTOPRISM_INIT` and set `PHOTOPRISM_DISABLE_CHOWN` to `"true"`:

https://docs.photoprism.app/getting-started/config-options/#docker-image

Note that Portainer also has a few quirks: It remembers previous defaults and applies them to future image versions, even if they use different defaults. This has led to errors in the past, but is beyond our control, so we cannot provide support in this case.

Also check out the troubleshooting checklists we provide:

https://docs.photoprism.app/getting-started/troubleshooting/#connection-fails

**Reply by MichaelLurie** [reply]
Well thanks for the input, but I waited now long enough and no change. So there must be another problem. Any other advice would be appreciated.

**Reply by lastzero** [reply]
See the troubleshooting checklist we provide specifically for this:

https://docs.photoprism.app/getting-started/troubleshooting/#connection-fails
